user_sync/connector/directory_ldap.py

```diff
@@ -179,7 +179,7 @@ def find_ldap_group_dn(self, group):
         group_filter_format = six.text_type(options['group_filter_format'])
         try:
             res = connection.search_s(base_dn, ldap.SCOPE_SUBTREE,
-                                      filterstr=group_filter_format.format(group=group), attrsonly=1)
+                                      filterstr=self.format_ldap_query_string(group_filter_format, group=group), attrsonly=1)
```

Comment:
Please pull out the `filterstr` argument calculation into a separate variable for readability:
```python
filter_string = self.format_ldap_query_string(group_filter_format, group=group)
res = connection.search_s(base_dn, ldap.SCOPE_SUBTREE, filterstr=filter_string, attrsonly=1)
```